lib/composer/wikia/fluent-sql-php/tests/FluentSql/InQueryTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace FluentSql;
+
+class InQueryTest extends FluentSqlTestBase {
+
+	public function testArrayInQuery() {
+		$expected = 'SELECT * FROM test_table WHERE test_column IN ( ?, ?, ? )';
+		$actual = (new SQL())
+			->SELECT_ALL()
+			->FROM('test_table')
+			->WHERE('test_column')
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "test_column" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)